saleor/graphql/order/tests/mutations/test_order_fulfill.py

```diff
@@ -1484,3 +1488,58 @@ def test_create_digital_fulfillment(
     get_graphql_content(response)
 
     assert mock_email_fulfillment.call_count == 1
+
+
+@patch("saleor.core.utils.events.transaction")
+@patch("saleor.plugins.webhook.plugin.trigger_webhooks_async")
+def test_order_fulfill_tracking_number_updated_event_triggered(
+    mocked_webhooks,
+    mocked_transactions,
+    any_webhook,
+    subscription_fulfillment_tracking_number_updated,
+    staff_api_client,
+    staff_user,
+    order_with_lines,
+    permission_group_manage_orders,
+    warehouse,
+    site_settings,
+    settings,
+):
+    # given
+    mocked_transactions.get_connection.return_value = MagicMock(in_atomic_block=False)
```

Comment:
Is there any better way to check if event was triggered? generally I couldn't catch event if it is wrapped in transaction block. This is workaround to see if event was triggered.

Reply:
Have you tried using `flush_post_commit_hooks`?

Reply:
looks like this solves the problem